ttnn/cpp/ttnn/tensor/tensor.hpp

```diff
@@ -298,22 +299,20 @@ struct Tensor {
 
 Tensor create_device_tensor(
     const ttnn::SimpleShape &logical_shape,
-    DataType dtype,
-    Layout layout,
-    Device *device,
-    const MemoryConfig &memory_config = {.memory_layout = tt::tt_metal::TensorMemoryLayout::INTERLEAVED},
-    const std::optional<Tile>& tile = std::nullopt);
+    const TensorLayout& layout,
+    Device *device);
 
+[[deprecated]]
 Tensor create_device_tensor(
-    const ttnn::SimpleShape &logical_shape,
-    const ttnn::SimpleShape &padded_shape,
+    const ttnn::SimpleShape &shape,
     DataType dtype,
     Layout layout,
     Device *device,
     const MemoryConfig &memory_config = {.memory_layout = tt::tt_metal::TensorMemoryLayout::INTERLEAVED},
     const std::optional<Tile>& tile = std::nullopt);
 
 // TODO: Remove once ALL ops switch over to return ttnn::SimpleShape in compute_output_shapes
+[[deprecated]]
```

Comment:
minor but we should add a string like your other examples. 

```
[[deprecated("Call create_device_tensor(...) with these arguments.........")]]
```

Reply:
Thank you!